translations/harbour-seaprint-de.ts

```diff
@@ -613,1234 +613,1201 @@ auf diesem Drucker</translation>
-        <translation type="unfinished"></translation>
+        <translation>13 x 19 Zoll</translation>
     </message>
     <message>
         <source>30 x 42″</source>
-        <translation type="unfinished"></translation>
+        <translation>30 x 42 Zoll</translation>
     </message>
     <message>
         <source>12 x 16″</source>
-        <translation type="unfinished"></translation>
+        <translation>12 x 16 Zoll</translation>
     </message>
     <message>
         <source>14 x 17″</source>
-        <translation type="unfinished"></translation>
+        <translation>14 x 17 Zoll</translation>
     </message>
     <message>
         <source>18 x 22″</source>
-        <translation type="unfinished"></translation>
+        <translation>18 x 22 Zoll</translation>
     </message>
     <message>
         <source>17 x 24″</source>
-        <translation type="unfinished"></translation>
+        <translation>17 x 24 Zoll</translation>
     </message>
     <message>
         <source>2 x 3.5″</source>
-        <translation type="unfinished"></translation>
+        <translation>2 x 3,5 Zoll</translation>
     </message>
     <message>
         <source>10 x 12″</source>
-        <translation type="unfinished"></translation>
+        <translation>10 x 12 Zoll</translation>
     </message>
     <message>
         <source>20 x 24″</source>
-        <translation type="unfinished"></translation>
+        <translation>20 x 24 Zoll</translation>
     </message>
     <message>
         <source>3.5 x 5″</source>
-        <translation type="unfinished"></translation>
+        <translation>3,5 x 5 Zoll</translation>
     </message>
     <message>
         <source>4 x 4″</source>
-        <translation type="unfinished"></translation>
+        <translation>4 x 4 Zoll</translation>
     </message>
     <message>
         <source>5 x 5″</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <source>184 x 260mm</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <source>195 x 270mm</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <source>55 x 85mm</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <source>55 x 91mm</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <source>54 x 86mm</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <source>275 x 395mm</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <source>89 x 119mm</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <source>Folio</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
+        <translation>5 x 5 Zoll</translation>
+    </message>-*
```

Comment:
What is the -* about?

Reply:
Maybe a mistake from me, Sorry. Please delete it. Or should I do it?